db/migrate/20211119152213_remove_hawkular_authentications.rb

```diff
@@ -0,0 +1,19 @@
+class RemoveHawkularAuthentications < ActiveRecord::Migration[6.0]
+  class Authentication < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled
+
+    belongs_to :resource, :polymorphic => true
```

Comment:
I don't see the resource being used in the migration, so do we need it?  If it's only for specs, I recommend just using resource_type, resource_id directly (which I think you already did)

Reply:
It made the specs a little cleaner (I could do kube.endpoints for example), I'll move the `belongs_to :resource, :polymorphic => true` to the spec file